src/site/content/en/blog/referrer-best-practices/index.md

```diff
@@ -346,16 +346,66 @@ available).
+Payment providers may rely on the `Referer` header of incoming requests for security checks. 
+
+For example: 
+
+- The user clicks a "Buy" button on *online-shop.example/cart/checkout*.
+- *online-shop.example* redirects to *payment-provider.example* to manage the transaction.
+- *payment-provider.example* checks the `Referer` of this request against a list of allowed
+  `Referer` values set up by the merchants. If it doesn't match any entry in the list,
+  *payment-provider.example* rejects the request. If it does match, the user can proceed to the
+  transaction.
+
+**Best practices for payment flow security checks, TL;DR: as a payment provider, you can use the
+`Referer` as a basic check against naive attacks—but you should absolutely have another, more
+reliable verification method in place.**
+
+The `Referer` header alone isn't a reliable basis for a check: the requesting site, whether they're
+a legitimate merchant or not, can easily set a `no-referrer` policy which will make the `Referer`
+information unavailable to the payment provider. However, as a payment provider, looking at the
+`Referer` may help you catch naive attackers who did not set a `no-referrer` policy. So you can
+decide to use the `Referer` as a first basic check. If you do so: 
+
+- **Do not expect the `Referer` to always be present; and if it's present, only check against the
+  piece of data it will include at the minimum: the origin**. When setting the list of allowed
+  `Referer`s, make sure that no path is included, but only the origin. Example:
+  *online-shop.example*'s allowed `Referer` should be *online-shop.example*, not
+  *online-shop.example/cart/checkout*. Why? Because by expecting either no `Referer` at all or a
+  `Referer` which value is the origin of the requesting website, you prevent unexpected errors since
+  you're **not making assumptions about the `Referrer-Policy`** your merchant has set or about the
+  browser's behavior if the merchant has policy is set. Both the site and the browser could strip
+  the `Referer` sent in the incoming request to just the origin or not send the `Referer` at all. 
+- If the `Referer` is absent or if it's present and your basic `Referer` origin check was
+  successful: you can move onto your other, more reliable verification method (see below).
+
+{% Aside %} If the merchant site uses HTTP and the payment provider site HTTPS, with the policies
+`strict-origin-when-cross-origin` and `no-referrer-when-downgrade` the `Referer` will not be sent at
+all. Since [most browsers use one of these policies by
+default](#default-referrer-policies-in-browsers), if the merchant site uses HTTP and has no policy
+set, no `Referer` will be visible from the HTTPS merchant site. This also means that the [Chrome
+change to a new default
+policy](https://developers.google.com/web/updates/2020/07/referrer-policy-new-chrome-default) won't
+change current behaviours for an HTTP merchant site with an HTTPS payment provider. {% endAside %}
```

Comment:
```suggestion
{% Aside %}
Suppose a merchant site uses HTTP and a payment provider uses HTTPS.
The payment provider is also using the `strict-origin-when-cross-origin` and
`no-referrer-when-downgrade` policies. In this scenario the browser will send
no `Referer` at all. Since [most browsers use one of these
policies by default](#default-referrer-policies-in-browsers), this means
that the [Chrome change to a new default
policy](https://developers.google.com/web/updates/2020/07/referrer-policy-new-chrome-default)
won't change current behavior.
{% endAside %}
```
FYI I struggled a lot with this aside. I had to go back and read the article to figure out what it was saying. I'm just providing this as rationale for why I'm proposing a rewrite (if I struggled with it, our audience might also struggle with it)

I think there's also a typo that significantly changes meaning but I didn't want to guess at what you meant:

> if the merchant site uses HTTP and has no policy
set, no `Referer` will be visible from the HTTPS merchant site

Is that supposed to be `will be visible from the HTTPS payment provider site`?

Reply:
Absolutely agree to rewrite, this is too complicated 👍 
Suggested and pushed a rewrite from my side, because 1. I see now that this paragraph needs a cleanup+reordering on top of the rewrite (and 2. The payment provider's referrer-policy doesn't play a role).
WDYT?